tutorials/assets_pipeline/importing_3d_scenes/node_type_customization.rst

```diff
@@ -0,0 +1,125 @@
+.. _doc_importing_3d_scenes_node_type_customization:
+
+Node type customization using name suffixes
+===========================================
+
+Many times, when editing a scene, there are common tasks that need to be done
+after exporting:
+
+- Adding collision detection to objects.
+- Setting objects as navigation meshes.
+- Deleting nodes that are not used in the game engine (like specific lights used
+  for modelling).
```

Comment:
```suggestion
  for modeling).
```

Reply:
There are more cases in this repo than just the importing 3D scenes page, so I opened another PR to fix the typos: https://github.com/godotengine/godot-docs/pull/8120. I can rebase this PR on top of that.

Reply:
Just merged it.